### ISSUE TITLE
Build: Translations are required on the French page

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#139]
URL:  [https://rsams.dev.cloud.openplus.ca/en/user/login](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Frsams.dev.cloud.openplus.ca%2Fen%2Fuser%2Flogin&data=04%7C01%7Cmanon.miron%40hc-sc.gc.ca%7Cec45cdb9ab0f4dfa788d08da02d39342%7C42fd9015de4d4223a368baeacab48927%7C0%7C0%7C637825404970600019%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=lo9rmCnhQp3Bhwvg5kSXBPsTI%2BEaKTdO2FReS2vb1Hs%3D&reserved=0)
User Agent: Chrome v99
Language: French and English

Issue Description: The text needs to match to that of the language of the page. For any text that is not going to be translated a lang attribute (lang="en") will be required.

Steps to reproduce: 
Test 1:
1. Log into the French version of the app using the url above
2. From the ‘Tableau de bord’ (Dashboard) page, select any of the titles under Titre and click on the ‘Générer’ button
3. The ‘Générer’ page has '(fr)' on both the English and French page under Workflow / Processus
4. Click on the 'Authlink' tab: The word 'View' needs to be translated 

Test 2:
1. Log into the French version of the app using the url above
2. From the ‘Tableau de bord’ (Dashboard) page, select any of the titles under Titre and click on the ‘Générer’ button
3. Select the ‘Revisions’ tab: The text ‘Use the radio buttons in the table below to select two revisions to compare. Then click the "Compare selected revisions" button to generate the comparison’ needs to be translated, as well as, ‘Select Revision A’ & ‘Select Revision B’. 

Expected results: The text should match the language of the page

![image](https://user-images.githubusercontent.com/68117281/160192254-0db2d66b-9370-47ac-abfa-089577631577.png)

![image](https://user-images.githubusercontent.com/68117281/160191854-d0bddc68-e134-4b0e-8987-462bcc9f0126.png)

![image](https://user-images.githubusercontent.com/68117281/160191799-0956b6dd-3cf1-4347-b977-b917dbdc70c8.png)

**Comment by echouin** [comment]
Hi @ndolter for the first two screenshots:
What is your Language Preference set to in your profile?
If you have it set to "no preference" you will view EN side w/ EN menus and when looking at FR pages, you will see FR menu items.

The Box on the right is telling you, while you are on the EN side, that you are missing brand values on the FR side of things. So, this appears to be working as intended.

As for the other items, I will follow-up w/ OP (note for Eric (Me), item in monday: https://openplus.monday.com/boards/1919962705/pulses/2474993127)

**Comment by ndolter** [comment]
Hi @echouin.  My language preference is set to 'No preference'.

**Comment by echouin** [comment]
@ndolter this is functioning as intended.

When viewing menus on the EN side (w/ either EN only or "no preference), you need to know that the item has an issue in FR "(FR)" side of the webpage.

When viewing the French webpage, it is confirming that the issue is in FR.

If I set my menu interface to FRENCH only, I would need this feature in order to be able to tell which side of the recall (EN or FR) is currently causing me an issue preventing it from going to the next stages of production.

You'll notice that it tells me I have an error in FR but the text "Brand" is user-interface, so it is in FR for me.

I'm seeing an issue when viewing menus in French however that i will need to follow-up w/ OP. But the Language issue is not there for that one.

**Comment by ndolter** [comment]
@echouin Thanks for the explanation.  : )
